extensions/wikia/Forum/ForumBoard.class.php

```diff
@@ -133,4 +134,12 @@ private function getLastPost( $rev_id ) {
 		}
 		return $postInfo;
 	}
+
+	/**
+	 * @param $forumBoardId integer
+	 * @return string cache key for getBoardInfo()
+	 */
+	private static function getBoardInfoCacheKey( $forumBoardId ) {
+		return implode( ':', [ self::FORUM_BOARD_INFO, self::CACHE_VERSION, $forumBoardId ] );
```

Comment:
Let's call `wfMemcKey` inside this method and return a full key to be used (including per wiki prefix) from this helper.